doc/changes/devel/13146.bugfix.rst

```diff
@@ -1 +1 @@
-Improved handling of problematic MFF files in `mne.io.read_raw_egi` by `Scott Huberty`_.
\ No newline at end of file
+Improved handling of problematic MFF files in `mne.io.read_raw_egi` by `Scott Huberty`_.
```

Comment:
Sorry - This change log is for #13145 .. I'm not sure how it ended up in this branch. So I deleted it from this branch in 68c118d ..

Reply:
.. And now I realize that means you have to add back the `no-changelog-needed` tag.. Sorry!!!!